### PR TITLE
Create riskman8.json

### PR DESCRIPTION
**Pull request by riskman8**
@githubteacher is my test.
this PULL closes #12758

### CODE REVIEW

File: _pins/riskman8.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: riskman8
```
Comment (NeutrinoX3) [inline _pins/riskman8.json]: @riskman8 You need to add `---` before this line and after the last line. Then you should be good to go 👍

File: _pins/riskman8.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+githubHandle: riskman8  
+latitude: 30.3793
+longitude: -90.9714
```
Comment (NeutrinoX3) [inline _pins/riskman8.json]: Add `---` after this line.

**Comment by ubet8932** [comment]
I have the same issues, for some checks are not successful. I am wondering how to fix it.

**Comment by crichID** [comment]
@riskman8 I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/12759/commits/02330904e19659cbf52eaa34c33863ce519854b2. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by December 29th, I will go ahead and do it just to keep the repository clean!

**Comment by pulidoo13** [comment]
Nice job 💯

**Pull request merged**